apiserver/params/deploy.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2021 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package params
+
+import "encoding/json"
+
+// DeployRequest describes a request to transactionally deploy a base bundle
+// which may be optionally accompanied by one or more overlay documents.
+type DeployRequest struct {
+	Options DeployOptions `json:"options"`
+
+	// A set of bundles to be deployed.
```

Comment:
I think this ticks off my linter because it does start with "Bundles is a...".

Reply:
There are some typos which I am fixing ATM but this one should not trigger the linter (it's usually checked for functions/methods/structs and not struct fields)